scanpipe/pipelines/deploy_to_develop.py

```diff
@@ -49,6 +49,7 @@ def steps(cls):
             cls.checksum_map,
             cls.java_to_class_map,
             cls.jar_to_source_map,
+            cls.javascript_match,
```

Comment:
this would be a map now, not a match... we renamed these. Matching is reserved for matching to the purldb, mapping is for d2d